### PR TITLE
Use c8 for test coverage

### PR DESCRIPTION
**Pull request by brunoSnoww**
This PR addresses https://github.com/josdejong/workerpool/issues/360. It replaces the ```istanbul``` test coverage package to [c8](https://github.com/bcoe/c8). It generates a .html report under ./coverage for all test files

### CODE REVIEW

**Comment by josdejong** [comment]
Thanks Bruno! 

When I run `npm run coverage` (on Windows), I get an error: 

```
Error: Cannot find module 'html;'
```

All works when I replace `c8 report --reporter=html; ` with `c8 report --reporter=html && `, can you adjust that in the PR?

**Comment by brunoSnoww** [comment]
thanks for pointing that out ! 😬

File: package.json [hunk 1]
```diff
@@ -30,23 +30,25 @@
...
     "gulp": "4.0.2",
     "handlebars": "4.7.7",
-    "istanbul": "0.4.5",
     "mocha": "10.2.0",
     "uglify-js": "3.17.4",
     "webpack": "5.76.0"
   },
-  "dependencies": {}
+  "dependencies": {
+    "commit": "^0.0.4"
```
Comment (josdejong) [inline package.json:52]: What is the new dependency `commit` needed for?

Comment (josdejong) [reply package.json:52]: I guess that is by mistake?

**Comment by josdejong** [comment]
Thanks, looks good to go Bruno 👌

**Pull request merged**